schemas/logs/aws/guard_duty.yml

```diff
@@ -0,0 +1,94 @@
+schema: AWS.GuardDuty
+parser:
+  native:
+    name: AWS.GuardDuty
```

Comment:
GuardDuty parser extracts some indicators e.g. AWS ARNs however this file doesn't declare any. Will the table in the datalake have these fields?

Reply:
The `native` parser config will force panther to resolve the schema from the `logtypes.Entry`. So yes, schemas in the data lake will be the same as they were. As we progress porting more 'native' log types to  either direct YAML-based defintions or non-legacy formats, the schema definitions here will be getting 'richer'.